### DISCUSSION TITLE
Browser monitoring for frontend applications not developed with Node.js

### DISCUSSION THREAD

**Discussion by dvisigalli** [apache/skywalking#9268] [Q&A] [answered]
Is possible in some way to activate browser monitoring for legacy Javascript applications that are not developed in Node.js?
I read the docs at https://github.com/apache/skywalking-client-js/tree/v0.8.0 but it seems to me that the only way to activate tracing is to have a Node.js application, install 'skywalking-client-js' with npm, importing the module and configuring a ClientMonitor instance.
Thanks!
@Fine0830

**Comment by Superskyyy** [comment, marked as answer]
I suppose you can simply load the bundled js file (lib/index.js) from a CDN and use it in plain JS applications, it doesn't have to rely on Node.js environment.
